components/combobox/dist/metadata.json

```diff
@@ -14,7 +14,6 @@
     ".spectrum-Combobox--quiet .spectrum-Combobox-textfield.is-readOnly.is-disabled .spectrum-Combobox-input:read-only",
     ".spectrum-Combobox--quiet .spectrum-Combobox-textfield.is-readOnly.is-invalid > .spectrum-Combobox-input:read-only",
     ".spectrum-Combobox--quiet.spectrum-Combobox--sizeL",
-    ".spectrum-Combobox--quiet.spectrum-Combobox--sizeM",
```

Comment:
I wonder if this snuck in during a rebase? I don't remember seeing a combobox diff before.